src/Document/CategoryManager.php

```diff
@@ -24,7 +24,7 @@ public function getPager(array $criteria, $page, $limit = 10, array $sort = [])
     {
         $parameters = [];
 
-        $query = $this->getRepository()
+        $query = $this->getDocumentManager()
```

Comment:
I think this would be something like:

```php
$query = $this->getDocumentManager()->createQueryBuilder();
$query->field('enabled')->equals($criteria['enabled'] ?? true);

$pager = new Pager();
$pager->setMaxPerPage($limit);
$pager->setQuery(new ProxyQuery($query));
// ...
```

Reply:
I added a new commit @franmomu: 9b188f5415668d14ad890ae4edc34dd3773f3eee.
Could you please check and confirm if this seems fine to you?
Is there any way this ever worked the way it was written before?
If no, I think we must provide a separate PR for these changes.
Thank you.

Reply:
Yeah, this looks fine and I would say it has never worked before, those methods did not exist in the odm 😕

Reply:
Fixed in #724.
Thank you so much.